src/components/atom/PoweredBy/index.tsx

```diff
@@ -0,0 +1,14 @@
+const PoweredBy = () => {
+  return (
+    <div className="bg-primitives-color-purple-purple-200 relative flex items-center justify-center py-[9px]">
+      <p className="text-tokens-label text-[13px] font-normal tracking-[0.20px] opacity-80">
+        <span className="text-[#000000e6]">Powered by |&nbsp;</span>
+        <a href="https://mvpworkshop.co/" rel="noopener noreferrer" target="_blank">
+          <span className="text-[#6331f5] underline">MVP Workshop</span>
```

Comment:
ovaj tekst bi trebao da ide u en.json i onaj gore za Powered by, dogovor je neki bio da svi tekstovi idu tamo, ako nesto nije onda je to jer nam je promaklo